test/NNRODE_tests.jl

```diff
@@ -0,0 +1,31 @@
+
+using Test, Flux, NeuralNetDiffEq, DifferentialEquations
+using DiffEqDevTools
+#Example 1
+function f(u,p,t,W)
+  2u*sin(W)
+end
+u0 = 0.00
+tspan = (0.0,5.0)
+prob = RODEProblem(f,u0,tspan)
+
+chain = Flux.Chain(Dense(2,5,σ),Dense(5,16 ,σ ) , Dense(16,2))
+opt = Flux.ADAM(0.1, (0.9, 0.95))
+sol = solve(prob, NeuralNetDiffEq.NNRODE(chain,opt), dt=1/20f0, verbose = true,
+            abstol=1e-10, maxiters = 1000)
```

Comment:
is there any good test we can add on the value, say doing it against Euler-Maruyama on the same noise process?

Reply:
Yes I can. But how do we compare them?

Reply:
Take a look at https://docs.juliadiffeq.org/latest/features/noise_process/ for how to directly work with a noise process. We'll want to reuse the same noise process between the two solves.

Reply:
Agreed. We can use alternate noise process. But how do we compare the results obtained from NNRODE and RandomEM?

Reply:
Instead of directly generating and using random numbers, generate a `NoiseGrid` https://docs.juliadiffeq.org/latest/features/noise_process/#NoiseGrid-2 and perform the fit against that, i.e. write the loss function so that it evaluates the loss using a full NoiseProcess. Then after the network is trained, we will have one solution, but also the trained neural network is stored in the alg. We can then solve RODEs with `RandomEM` and then use sol.W to grab the noise process and throw that in to the NN. If trained well, the difference between the two solutions will be small.

Reply:
Yes. This has been done in the latest commit. Can you please review this?